README.md

```diff
@@ -29,18 +29,18 @@ information:
 Additionally, the epoch for a Nanoflake is defined by the context in which
 it is generated and used.  That means that users are free to define their
 own epoch for their own applications, and use said epoch across all generators
-in order to ensure that nanoflakes are globally unique in their application.
+in order to ensure that nanoflakes are unique in their application.
 
 Since nanoflakes contain a timestamp, they can be sorted by time, and
 therefore can be used as a unique ID for a time series. A user can also
 use the nanoflake's timestamp to determine when the nanoflake was generated.
 This is done by shifting the timestamp 22 bits to the right, and adding the
 epoch to the result in order to get the nanoflake's timestamp in milliseconds
-since the UNIX epoch.
+since the epoch start as defined by the application.
```

Comment:
The previous version was actually correct. The timestamp is local to the epoch, so adding the epoch to the result gives a UNIX time in millis.

This paragraph should be either reverted back to how it was, or rewrite it as to remove "[...] and adding the
epoch to the result in order to get [...]"

Reply:
My mistake; it should be correct now. Thank you for the review!